src/rpcblockchain.cpp

```diff
@@ -384,6 +384,7 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"nonce\" : n,           (numeric) The nonce\n"
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
+            "  \"chainwork\" : \"xxxx\",  (string) The estimated number of block header hashes checked from the genesis block to this block, in hexadecimal\n"
```

Comment:
> block header hashes

is not immediately clear what this means. Usually blocks with hashes that don't hit the diff are not considered valid, so they are not counted. Then "estimated" gives the hint that they are counted as well.